submission/persistence-mongo/src/test/kotlin/ac/uk/ebi/biostd/persistence/doc/db/converters/from/DocSectionConverterTest.kt

```diff
@@ -0,0 +1,170 @@
+package ac.uk.ebi.biostd.persistence.doc.db.converters.from
+
+import ac.uk.ebi.biostd.persistence.doc.db.converters.to.CommonsConverter.classField
+import ac.uk.ebi.biostd.persistence.doc.model.*
+import arrow.core.Either
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.bson.Document
+import org.junit.jupiter.api.Test
+
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(MockKExtension::class)
+internal class DocSectionConverterTest(
+    @MockK val docAttributeConverter: DocAttributeConverter,
+    @MockK val docAttribute1: DocAttribute,
+    @MockK val docAttribute2: DocAttribute,
+    @MockK val docAttribute3: DocAttribute,
+
+    @MockK val attributeDocument1: Document,
+    @MockK val attributeDocument2: Document,
+    @MockK val attributeDocument3: Document,
+
+    @MockK val docFileListConverter: DocFileListConverter,
+    @MockK val docFileList1: DocFileList,
+    @MockK val docFileList2: DocFileList,
+    @MockK val fileListDocument1: Document,
+    @MockK val fileListDocument2: Document,
+
+    @MockK val docFileConverter: DocFileConverter,
+    @MockK val docFile1: DocFile,
+    @MockK val fileDocument1: Document,
+
+    @MockK val docFileTableConverter: DocFileTableConverter,
+    @MockK val docFileTable1: DocFileTable,
+    @MockK val fileTableDocument1: Document,
+
+    @MockK val docLinkConverter: DocLinkConverter,
+    @MockK val docLink1: DocLink,
+    @MockK val linkDocument1: Document,
+
+    @MockK val docLinkTableConverter: DocLinkTableConverter,
+    @MockK val docLinkTable1: DocLinkTable,
+    @MockK val linkTableDocument1: Document
+) {
+    private val testInstance = DocSectionConverter(
+        docAttributeConverter,
+        docLinkConverter,
+        docLinkTableConverter,
+        docFileConverter,
+        docFileTableConverter,
+        docFileListConverter
+    )
+
+    @Test
+    fun convert() {
+        every { docAttributeConverter.convert(attributeDocument1) } returns docAttribute1
```

Comment:
In order to avoid having so many mock definitions in the class definition, you should declare them in the method where it's actually used. For example:

```
@Test
fun convert(@MockK docAttribute: DocAttribute) {
     every { docAttributeConverter.convert(attributeDocument1) } returns docAttribute
     ...
}
```

Reply:
I do not see a big difference given that there is a single test methods